### ISSUE TITLE
Fix contrast for network manager checklist when items are complete

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#3403]
Once items are checked off, the contrast changes to a severe WCAG failure. We should change this so the state is still clear but the text remains readable.

Element has insufficient color contrast of 3.24 (foreground color: #727272, background color: #d4d4d4).

Foreground color of #3F3F3F would be sufficient here for both WCAG AA (4.5:1) and WCAG AAA (7:1).

_Originally posted by @greatislander in https://github.com/pressbooks/pressbooks-network-analytics/issues/316#issuecomment-1629244722_
